test/dogma/formatter/emacs_test.exs

```diff
@@ -0,0 +1,74 @@
+defmodule Dogma.Formatter.EmacsTest do
+  use ShouldI
+
+  alias Dogma.Formatter.Emacs
+  alias Dogma.Script
+  alias Dogma.Error
+
+  with ".start" do
+    with "no files to list" do
+      should "print nothing" do
+        assert Emacs.start([]) == ""
+      end
+    end
+
+    with "some files to lint" do
+      should "print nothing" do
+        script = [%Script{}, %Script{}, %Script{}]
+        assert Emacs.start(script) == ""
+      end
+    end
+  end
+
+  with ".script" do
+    with "no errors" do
+      should "print nothing" do
+        script = %Script{ errors: [] }
+        assert Emacs.script(script) == ""
+      end
+    end
+
+    with "some errors" do
```

Comment:
It's a parallel test case to https://github.com/lpil/dogma/blob/master/test/dogma/formatter/simple_test.exs#L39. In the `.script` function it doesn't seem like formatters really care about the contents of the error, just if they should note an error has happened.